charts/clusternet-agent/README.md

```diff
@@ -21,6 +21,7 @@ helm install clusternet-agent -n clusternet-system --create-namespace \
   --set parentURL=PLEASE-CHANGE-ME \
   --set registrationToken=PLEASE-CHANGE-ME \
   --set extraArgs.cluster-reg-name=PLEASE-CHANGE-ME \
+  --set extraArgs.cluster-reg-namespace=PLEASE-CHANGE-ME \
```

Comment:
This is optional. We should make it like a required parameter.

Reply:
should not?

Reply:
Yeah, better not make it mandatory. This would mislead users.

Reply:
ok, I'll remove it